src/main/scala/server/Server.scala

```diff
@@ -59,7 +59,8 @@ class Server(
   akkaLog: String => ingredients.logging.PlainOldLogger,
   config: ServerConfig,
   router: akka.actor.ActorRefFactory => RequestContext => Unit,
-  akkaAdditionalConf: com.typesafe.config.ConfigMergeable = ConfigFactory.empty) {
+  akkaAdditionalConf: com.typesafe.config.ConfigMergeable = ConfigFactory.empty,
+  bootTimeout: duration.Duration = duration.Duration(5, duration.SECONDS)) {
```

Comment:
instead of repeating the default duration twice, can you extract it to a (private) variable and use it?